<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background-image: url(&quot;../TextEditor_GUI_Bonus/te.png&quot;);

}

QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(227, 226, 219);
border: 0px solid black;
background-color: rgb(46, 82, 102);
border-radius: 7px;
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border: 0.5px solid black;
border-radius: 7px;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addcontent_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
border: 0px solid black;
background-color: rgb(46, 82, 102);
border-radius: 7px;
}
</string>
    </property>
    <property name="text">
     <string>Add Content</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="erase_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>54</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
border: 0px solid black;
background-color: rgb(46, 82, 102);
border-radius: 7px;
}</string>
    </property>
    <property name="text">
     <string>Erase Content</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="encrypt_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>98</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Encrypt Content</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>142</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Decrypt Content</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="findword_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
border: 0px solid black;
background-color: rgb(46, 82, 102);
border-radius: 7px;
}
</string>
    </property>
    <property name="text">
     <string>Find Word</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>186</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Merge Content</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lowercase_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>362</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Lower Case</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="uppercase_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>318</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Upper Case</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cap_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>406</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Capitalize</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="repeatedword_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>274</y>
      <width>189</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>No. Repeated Word</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lines">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>475</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>475</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lines</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>475</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Words</string>
    </property>
   </widget>
   <widget class="QLabel" name="words">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>475</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="chars">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>475</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>475</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Characters</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>473</y>
      <width>541</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
border: 0.5px solid black;
border-radius: 7px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
background-image: url(&quot;../TextEditor_GUI_Bonus/te.png&quot;);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="welcomewidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">



</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>361</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(46, 82, 102);
font: 700 23pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>Text Editor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>441</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(46, 82, 102);
font: 19pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>Ahlan ya user ya habibi</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>350</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 700 14pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
border-radius: 11px;
}
</string>
      </property>
      <property name="text">
       <string>Load File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>441</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(46, 82, 102);
font: 16pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>Choose file to edit</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="createnew_btn">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>350</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 700 14pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
border-radius: 11px;
}
</string>
      </property>
      <property name="text">
       <string>Create New File</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="apply_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_btn_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 10pt &quot;Georgia&quot;;
color: rgb(211, 208, 203);
background-color: rgb(46, 82, 102);
}
</string>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>473</y>
      <width>2</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>473</y>
      <width>2</width>
      <height>15</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>widget</zorder>
   <zorder>find_btn_2</zorder>
   <zorder>apply_btn</zorder>
   <zorder>addcontent_btn</zorder>
   <zorder>findword_btn</zorder>
   <zorder>save_btn</zorder>
   <zorder>repeatedword_btn</zorder>
   <zorder>cap_btn</zorder>
   <zorder>decrypt_btn</zorder>
   <zorder>uppercase_btn</zorder>
   <zorder>lowercase_btn</zorder>
   <zorder>erase_btn</zorder>
   <zorder>encrypt_btn</zorder>
   <zorder>merge_btn</zorder>
   <zorder>textEdit</zorder>
   <zorder>lines</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>words</zorder>
   <zorder>chars</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit</zorder>
   <zorder>find_btn</zorder>
   <zorder>frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>findword_btn</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findword_btn</sender>
   <signal>clicked()</signal>
   <receiver>find_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>find_btn</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>find_btn</sender>
   <signal>clicked()</signal>
   <receiver>find_btn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addcontent_btn</sender>
   <signal>clicked()</signal>
   <receiver>apply_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apply_btn</sender>
   <signal>clicked()</signal>
   <receiver>apply_btn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repeatedword_btn</sender>
   <signal>clicked()</signal>
   <receiver>find_btn_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repeatedword_btn</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>find_btn_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>find_btn_2</sender>
   <signal>clicked()</signal>
   <receiver>find_btn_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
